configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionStateWatcher.java

```diff
@@ -48,6 +48,7 @@ private synchronized void sessionStatusChanged(Status newStatus) {
         switch (newStatus) {
             case NEW:
             case NONE:
+            case DELETE:
```

Comment:
Doing nothing on DELETE means we must be sure the DEACTIVATE has already been acted upon, and the session deactivated. How can we be sure this is the case here?

Reply:
Thanks for reminding me, that was then one thing I was thinking "that needs to be done as well", but had forgotten about today.

Reply:
Fixed